docs/getting-started/architecture/feature-transformation.md

```diff
@@ -115,4 +115,72 @@ feature_view = FeatureView(
     ...
 )
 ```
-The underlying implementation of the join is an inner join by default, and join key is the entity id.
\ No newline at end of file
+The underlying implementation of the join is an inner join by default, and join key is the entity id.
+
+### Tiling
+
+Tiling is a transformation pattern designed for efficient streaming feature engineering, inspired by Chronon's tiled architecture. It divides time-series data into manageable chunks (tiles) for processing temporal aggregations and derived features efficiently. This is particularly useful for:
+
+- Temporal aggregations over sliding windows
+- Chaining features across different time horizons  
+- Handling late-arriving data in streaming scenarios
+- Memory-efficient processing of large time-series data
+
+Examples include:
+
+```python
+from feast.transformation import tiled_transformation
+from datetime import timedelta
+
+@tiled_transformation(
+    tile_size=timedelta(hours=1),  # Process data in 1-hour tiles
+    mode="pandas",                 # Use pandas processing mode
+    overlap=timedelta(minutes=5),  # 5-minute overlap between tiles
+    aggregation_functions=[
+        lambda df: df.groupby('entity_id').agg({
+            'transaction_amount': ['sum', 'mean', 'count']
+        }).reset_index()
+    ]
+)
+def hourly_transaction_features(df: pd.DataFrame) -> pd.DataFrame:
+    """Transform transaction data within each hour tile."""
+    return df.assign(
+        rolling_avg=df['transaction_amount'].rolling(window=10).mean(),
+        cumulative_amount=df['transaction_amount'].cumsum()
+    )
+
+# Usage in StreamFeatureView
+stream_fv = StreamFeatureView(
```

Comment:
It doesn't make sense to pass in a pandas data frame transformation into the Stream Feature View, does it? I suppose it may, we support the "spark" mode today. But this isn't compatible  with it as the "mode" is passed into the tiled_transformation vs the StreamFeatureView, right?